Update README (#121072)
microsoft/winget-pkgs: README.md

[![Publish Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=PublishPipeline 'Publish Pipeline Badge')](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338)
[![GitHub Status](https://img.shields.io/endpoint?url=https://api.bittu.eu.org/github-status-badge-endpoint)](https://www.githubstatus.com)

> This repository contains the manifest files for the **Windows Package Manager** default source. To browse the available packages, you can for example use the third-party service [winstall](http://winstall.app).

Reviewer: ```suggestion
This repository contains the manifest files for the **Windows Package Manager** default source.
```

> This repository contains the manifest files for the **Windows Package Manager** default source. To browse the available packages, you can for example use the third-party service [winstall](http://winstall.app).

Reviewer: ```suggestion
This repository contains the manifest files for the **Windows Package Manager** default source. To browse the available packages, you can use the third-party services listed in [THIRD_PARTY.md](https://github.com/microsoft/winget-pkgs/blob/master/THIRD_PARTY.md).
```

The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).

# Submitting a Package

> You are highly encouraged to submit manifests for your favorite application.
> 
> > [!IMPORTANT]
> > At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
> 
> To submit a package to this repository, you should follow these steps:

Reviewer: ```suggestion
```
